### PR TITLE
added new science fiction book

### PR DESCRIPTION
**Pull request by YashAmin-01**
(todo:please remove this todo and the sections you did not check below before you make your pull request.
If you are unsure, please check other PRs like this one: https://github.com/hackerkid/Mind-Expanding-Books/pull/207#issue-377268434)

## In this pull request 
- [x]  I am adding a new book.

## Adding new book
* The book I am adding is "The time machine" by H.G Wells
* I am adding this book to Science Fiction section.
* I have read this book 1 times.
* I think this book deserves to be in this list because it has a different perspective towards time travel. Most time travel stories show flying cars and tall buildings but in this the writer shows a different perspective where humans start living peacefully and in harmony in the distant future and perspective of the time traveler who observes all these changes in the human race.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Mind Expanding Books
```
Comment (hackerkid) [inline README.md:1]: Do you what is this change?

**Review by hackerkid — COMMENTED** [review]
Thanks for working on this. Added a minor comment.

File: README.md (hunk 1 above)
Comment (YashAmin-01) [reply README.md:1]: Hello, what changes do I have to make?

Comment (YashAmin-01) [reply README.md:1]: please accept my pr, i want to make progress in hacktoberfest

Comment (hackerkid) [reply README.md:1]: @YashAmin-01 I mean, do you know why this line has changed? Can you undo the change?

Comment (YashAmin-01) [reply README.md]: made the correction that you mentioned.

**Comment by hackerkid** [comment]
Thanks a lot for working on this. Merged thanks :)

**Comment by YashAmin-01** [comment]
Thank you for being so helpful :)
I would like to some more books that I have read.

**Comment by hackerkid** [comment]
@YashAmin-01 Sure. Go ahead!

**Pull request merged**